src/focus_manager/txm_focus_manager.js

```diff
@@ -262,7 +262,7 @@ export class TXMFocusManager {
         }
 
         let focus = this.currentFocus;
-        if (!focus) {
+        if (!focus && inputActions.isMovementAction(action)) {
```

Comment:
as discussed, we should also allow the action == inputActions.select as well

Reply:
Yep, just pushed the change 👍